### ISSUE TITLE
start / stop events

### ISSUE THREAD

**Issue by macarron** [Shoety/cordova-plugin-inappbrowserxwalk#7]
Hi,

I have some trouble with "start" and "stop" events. Sometimes the stop event fire before the start event. The only thing I could tell is my website navigation is in ajax...
The start event fire with all loaded ressources, not only the webpage. A way to get only the url web page is to dispatch the view url like that

```
obj.put("url", view.getUrl());
```

I think there is many work to do to improve this...

**Comment by ClaasBrueggemann** [comment]
You are right, the events are called for every ressource getting loaded. If i have time, i  will implement something like pageLoadStarted and pageLoadStopped. Meanwhile you can try this:

``` js
var ressourcesLoaded= 0;
browser.addEventListener("loadstart", function ( url ) {
  if (ressourcesLoaded === 0) {
     console,log("loadStarted");
  }
  ressourcesLoaded++;
});

browser.addEventListener("loadstop", function ( url ) {
   ressourcesLoaded--;
   if( ressourcesLoaded === 0) {
     console,log("loadStopped");
 }
});

```

**Comment by macarron** [comment]
Thank you I've already done something like that ;)

**Comment by macarron** [comment]
Bug fixed, is the same issue as take picture from camera. We have to use XWalkUIClient to listen view event instead of XWalkResourceClient.

**Comment by ClaasBrueggemann** [comment]
Ok please create another pull request for that. On the weekend i check both and merge them into master.
